databus_client/filters/filter.json

```diff
@@ -1 +1 @@
+{"matched_filter": "reject", "unmatched_filter": "accept", "non_metric_filter": [{"source": "CKRtEgdEUjNITU1P", "entity_type": "applications", "entityIds": ["13988:561:2096623162362547537", "v2"], "entityNames": ["n1", "n2"]}, {"source": "CNeJARIHRDRBR1U5TQ==", "entity_type": "applications", "entityNames": ["DATABUS_APP_TEST_01"]}, {"source": "SOURCE_TWO", "entity_type": "applications", "entityIds": ["v1", "v3"], "entityNames": ["n1", "n3"]}, {"source": "SOURCE_TWO", "entity_type": "applications", "entityIds": ["v1", "v3"], "entityNames": ["n1", "n3"]}], "metric_filter": [{"source": "CJWMARIHRFJHRkRYTw==", "entity_type": "Host", "metrics": ["net.usage.delta.summation.kiloBytes", "two"], "entityIds": ["17941:4:3519141807154183312", "v2"]}, {"source": "SOURCE-ONE", "entity_type": "VirtualMachine", "metrics": ["cpu.usage.rate.average.percent", "three"], "entityIds": ["v7", "v6"]}, {"source": "SOURCE_TWO", "entity_type": "Hosts", "metrics": ["cpu.usage.rate.average.percent", "three"], "entityIds": ["v1", "v3"]}], "sub_metric_filter": [{"source": "SOURCE-ONE", "entity_type": "vms-metrics", "metrics": ["cpu.usage.rate.average.percent", "two"], "entityIds": ["v1", "v2"]}, {"source": "SOURCE-ONE", "entity_type": "nsxt-edge-node-metrics", "metrics": ["edge.dataplane.cpu.core.usage.rate.average.percent", "two"], "entityIds": ["nsxten1", "nsxten2"]},{"source": "CJBOEgdEUkdCQUVT", "entity_type": "flows-metrics", "metrics": ["cpu.usage.rate.average.percent", "flow.totalBytes.delta.summation.bytes"], "entityIds": ["10000:515:3516501221883531266", "10000:515:7484076144344179859", "10000:515:1823077571592235831"]}]}
```

Comment:
Have you verified filters endpoint too?

Reply:
screenshots for filters endpoint verification added